.storybook/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+const path = require('path');
+
+module.exports = {
+  module: {
+    loaders: [
+      {
+        test: /css$/,
+        loaders: [ 'style', 'css' ],
+        include: path.resolve(__dirname, '../')
+      }
+    ]
+  },
```

Comment:
I'll leave that to another PR.  We can probably just leave it though, since we might want to add stylesheets later and it isn't hurting anything.